bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.html

```diff
@@ -1,37 +1,36 @@
-<ng-template #triggerSequenceButton>
-  <div>
-    <a (click)="navigateToTriggerSequence()">
-      <button dt-button uitestid="keptn-trigger-button-open">
-        <dt-icon name="flash"></dt-icon>
-        Trigger a new sequence
-      </button>
-    </a>
-  </div>
-</ng-template>
-
-<div fxFlexFill>
-  <div class="container p-0" fxFlex="40" fxLayout="column" fxLayoutGap="15px" *ngIf="project$ | async as project">
+<div fxFlexFill *ngIf="state$ | async as state">
+  <div class="container p-0" fxFlex="40" fxLayout="column" fxLayoutGap="15px">
+    <ng-template #triggerSequenceButton>
+      <div>
+        <a (click)="navigateToTriggerSequence(state.projectName)">
+          <button dt-button uitestid="keptn-trigger-button-open">
+            <dt-icon name="flash"></dt-icon>
+            Trigger a new sequence
+          </button>
+        </a>
+      </div>
+    </ng-template>
```

Comment:
After clicking through the Sequence screen, I noticed that the list of sequences is not stretching vertically to 100% height. Previously it would use the full height. I don't see actually which change might have caused that, maybe it's because of moving the `*ngIf` to the outer container? Can you try to reproduce it and fix it?
![image](https://user-images.githubusercontent.com/6098219/175282112-118bd819-f29c-4e4b-bb51-b6bd1c5f09ee.png)

Reply:
I removed `encapsulation: ViewEncapsulation.None,` and then the style in the related scss file wasn't applied anymore.
Added `:host::ng-deep` to it 👍